sematic/api/endpoints/meta.py

```diff
@@ -0,0 +1,52 @@
+"""Metadata about the server itself."""
+
+# Standard
+from typing import Optional
+
+# Third-party
+import flask
+
+# Sematic
+from sematic.api.app import sematic_api
+from sematic.versions import CURRENT_VERSION, MIN_CLIENT_SERVER_SUPPORTS
+from sematic.db.models.user import User
+from sematic.user_settings import MissingSettingsError, SettingsVar, get_user_settings
+from sematic.api.endpoints.auth import authenticate
+
+
+@sematic_api.route("/api/v1/meta/versions", methods=["GET"])
+def get_server_version_info() -> flask.Response:
+    """Get information about the server version and clients it supports
+
+    Returns
+    -------
+    A flask response with a json payload. The payload has two fields:
+        server: the version of Sematic the server is running, as 3 integers. Note that
+            clients whose version is greater than this may not work with the server.
+        min_client_supported: the minimum version of Sematic the current server version
+            supports. Clients with versions less than this may fail when used with the
+            server.
+    """
+    payload = dict(
+        server=CURRENT_VERSION,
+        min_client_supported=MIN_CLIENT_SERVER_SUPPORTS,
+    )
+
+    return flask.jsonify(payload)
+
+
+@sematic_api.route("/env", methods=["GET"])
```

Comment:
Do we need a `TODO` to move this to `/api/v1/meta/env` to align with the other endpoint in this module?